packages/cadl-typescript/test/commands/gen-cadl-ranch.ts

```diff
@@ -1,13 +1,13 @@
 import { cadls } from "./cadl-ranch-list.js";
```

Comment:
The filename `gen-cadl-ranch` should be `gen-typespec-ranch`. Is it changed this time or next time?